tethys_cli/proxyapps_commands.py

```diff
@@ -4,188 +4,167 @@
     write_info,
 )
 
-from pathlib import Path
+from tethys_cli.cli_helpers import setup_django
 
-from sqlalchemy import create_engine, exc
-from sqlalchemy.ext.automap import automap_base
-from sqlalchemy.orm import sessionmaker
-from tethys_apps.utilities import (
-    get_tethys_home_dir,
-)
-from .settings_commands import read_settings
-
-TETHYS_HOME = Path(get_tethys_home_dir())
+from django.db import IntegrityError
 
 
 def add_proxyapps_parser(subparsers):
     # Setup list command
     proxyapps_parser = subparsers.add_parser(
         "proxyapp", help="Add proxy apps and list proxy apps into the Tethys Platform"
     )
-    proxyapps_parser.add_argument(
-        "-l",
-        "--list",
-        help="list available proxy apps in the current tethys installation",
-        action="store_true",
+    proxyapps_subparsers = proxyapps_parser.add_subparsers(title="Options")
+
+    proxyapps_list_parser = proxyapps_subparsers.add_parser(
+        "list", help="list available proxy apps in the current tethys installation"
```

Comment:
Add a `-v,--verbose` option that will list all attributes of each proxy app.

Reply:
Done